sync-online/index.js

```diff
@@ -0,0 +1,66 @@
+// Our textarea
+var textarea = document.querySelector('textarea');
+
+// Register the ServiceWorker
```

Comment:
I think there's no need for all the extra checks here, fetch events are delayed if the service worker is activating.